core/src/banking_stage/unprocessed_transaction_storage.rs

```diff
@@ -139,6 +143,7 @@ pub struct ConsumeScannerPayload<'a> {
 }
 
 fn consume_scan_should_process_packet(
+    invalid_fee_payer_filter: &InvalidFeePayerFilter,
```

Comment:
nit: typically new args go at the end

Reply:
yeah, not sure what I was thinking: b268af7fc10f9c999ddf20fbec7ee382ece8663e